src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.max;
+
+public class Cars {
+    private final List<Car> carList;
+
+    public Cars() {
+        carList = new ArrayList<>();
+    }
```

Comment:
저는 이 방식과 성능차이가 없다고 생각이 들면서 가독성이 더 좋다고 생각이 드는데 이렇게 작성하신 이유가 있으신가요? ☺️

```suggestion
    private final List<Car> carList = new ArrayList<>();
```

Reply:
> 저는 이 방식과 성능차이가 없다고 생각이 들면서 가독성이 더 좋다고 생각이 드는데 이렇게 작성하신 이유가 있으신가요? ☺️

객체가 지연로딩된다는  장점이있을것 같긴하네요. 
가독성과의 트레이드오프의 부분이 있을것같긴하지만 객체의 생성에 그렇게 큰 비용이 드는 상황이 아니기때문에  가독성을 해칠만큼 지연로딩으로 얻는 이점이 크진않을것같아요
다른분들 의견도 궁금해지네요

Reply:
네 저도 무의식적으로 생성자로 먼저 생성한 다음 ArrayList를 할당하는 것이 낫지 않을까 생각했었는데 가독성 좋은 방식으로 작성하는 것이 낫겠네요 :)